lms/envs/common.py

```diff
@@ -4745,6 +4748,10 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
         ENTERPRISE_FULFILLMENT_OPERATOR_ROLE,
         ENTERPRISE_SSO_ORCHESTRATOR_OPERATOR_ROLE,
     ],
+    SYSTEM_ENTERPRISE_PROVISIONING_ADMIN_ROLE: [
+        PROVISIONING_ENTERPRISE_CUSTOMER_ADMIN_ROLE,
+        PROVISIONING_PENDING_ENTERPRISE_CUSTOMER_ADMIN_USER_ROLE,
```

Comment:
How about "PROVISION_ENTERPRISE_CUSTOMER_ROLE" and "PROVISION_ENTERPRISE_CUSTOMER_ADMIN_ROLE". This way, the name includes the exact feature that can be assigned to a user for the right permission.